libraries/AP_GyroFFT/AP_GyroFFT.cpp

```diff
@@ -166,6 +166,14 @@ const AP_Param::GroupInfo AP_GyroFFT::var_info[] = {
     // @RebootRequired: True
     AP_GROUPINFO("NUM_FRAMES", 14, AP_GyroFFT, _num_frames, 0),
 
+    // @Param: PFILT_MASK
```

Comment:
maybe an _OPTIONS bitmask (32 bits), with an enable bit for post filter?

Reply:
Renamed to FFT_OPTIONS